examples/demo-dts/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:18 as base
```

Comment:
Can we update base node images? Current [active LTS version](https://nodejs.org/en/about/previous-releases) is 22